ONSdigital/dp-frontend-renderer
# COVID-19 banner and restructure

### What

- COVID-19 banner is implemented as a partial.
- As we have multiple banners now, I've restructured the `partials` directory so that there is a `banners` subdirectory. In this subdirectory is where the `covid.tmpl` and `cookies.tmpl` banners live
- Renamed `cookies-banner` to `cookies` now that it resides in the new `banners` subdirectory. Updated the reference to the partial in `main` accordingly
- Added the COVID-19 banner to the previously empty `homepage.tmpl`
- Added the English and Welsh localisations from Babbage

![image](https://user-images.githubusercontent.com/23668262/80375028-80c02b80-888f-11ea-823d-0f685e9572e4.png)

### How to review

- Pull branch `feature/new-banner-styling` from Sixteens - use this for local testing
- Set `NEW_HOMEPAGE_ENABLED=true` on in the `dp-frontend-router` app
- Ensure `dp-frontend-homepage-controller` is running
- Check that the banner aligns with the design spec

### Who can review

Anyone but me
